packages/ckeditor5-emoji/tests/emojimention.js

```diff
@@ -195,6 +196,162 @@ describe( 'EmojiMention', () => {
 		} );
 	} );
 
+	it( 'should set emoji mention feed configuration only once', async () => {
+		const editorElement = document.createElement( 'div' );
+		const editor1Element = document.createElement( 'div' );
+		document.body.appendChild( editorElement );
+
+		const editor = await ClassicTestEditor.create( editorElement, {
+			plugins: [ EmojiMention, Mention ],
+			substitutePlugins: [ EmojiDatabaseMock ]
+		} );
+
+		const editor1 = await ClassicTestEditor.create( editorElement, {
```

Comment:
```suggestion
		const editor1 = await ClassicTestEditor.create( editor1Element, {
```

Reply:
Fixed in [0e071f6](https://github.com/ckeditor/ckeditor5/pull/17826/commits/0e071f61f260d4c85e2311852622560bae43f695)